Sources/protoc-gen-grpc-swift/main.swift

```diff
@@ -65,14 +65,9 @@ func outputFileName(
   component: String,
   fileDescriptor: FileDescriptor,
   fileNamingOption: FileNaming,
-  binaryFile: Bool
+  extensionType: String
```

Comment:
Just `extension` is fine

Reply:
I tried, but since it is a keyword it doesn't work

Reply:
would fileExtension be better?

Reply:
You can wrap it in backticks